### DISCUSSION TITLE
Implementing unique features not documented in Wails - Inputs?

### DISCUSSION THREAD

**Discussion by ha-sante** [wailsapp/wails#1787] [Q&A]
I would like to implement a feature to

1. Run our product at Startup
2. Have the product stay in System Tray when close is clicked (Exit when right click, to close)

Already seen [this ](https://github.com/getlantern/systray) for system tray 

Any pointers for a solution to Running the product on Startup?

**Comment by leaanthony** [comment]
https://github.com/wailsapp/wails/issues/1789
https://github.com/wailsapp/wails/issues/1521
👍
